code/modules/projectiles/updated_projectiles/gun_helpers.dm

```diff
@@ -326,11 +294,10 @@ should be alright.
 			if(current_mag)
 				unload(user,0,1)
 				to_chat(user, "<span class='notice'>You start a tactical reload.</span>")
-			var/old_mag_loc = AM.loc
 			var/tac_reload_time = 15
 			if(user.mind && user.mind.cm_skills)
 				tac_reload_time = max(15 - 5*user.mind.cm_skills.firearms, 5)
-			if(do_after(user,tac_reload_time, TRUE, 5, BUSY_ICON_FRIENDLY) && AM.loc == old_mag_loc && !current_mag)
```

Comment:
Is the removal of the icon intended here?

Reply:
delay is very short and no visible message.